WalletWasabi/Services/IndexDownloader.cs

```diff
@@ -119,32 +122,31 @@ public void Synchronize(TimeSpan requestInterval)
 							// If stop was requested return.
 							if (IsRunning == false) return;
 
-							FilterModel bestKnownFilter;
 							using (await IndexLock.LockAsync())
 							{
 								bestKnownFilter = Index.Last();
 							}
 
-							var response = await Client.SendAsync(HttpMethod.Get, $"/api/v1/btc/blockchain/filters?bestKnownBlockHash={bestKnownFilter.BlockHash}&count=1000");
+							var filters = await WasabiClient.GetFiltersAsync(bestKnownFilter.BlockHash, 1000);
 
-							if (response.StatusCode == HttpStatusCode.NoContent)
+							if (filters != null && !filters.Any())
```

Comment:
`GetFiltersAsync` does never return null, you don't need to check for null.

Reply:
Removed.